docs/source/getting_started/quick_start.md

```diff
@@ -0,0 +1,364 @@
+# Quick Start
+
+In this tutorial, we will: 
+* Write a workflow to sort and assemble COVID genomes
+* Customize a front-end interface for our workflow
+* Define the necessary compute and resource requirements
+* Upload the workflow onto Latch
+* Test our workflow from a user-friendly interface 
+
+After going through the tutorial, you should have a conceptual understanding of how to translate a set of command lines into a Latch workflow and how to generate an interactive interface for that workflow. 
```

Comment:
"you should have a conceptual understanding of how to translate a set of command lines into a Latch workflow" -> "you should have a strong understanding of how to translate a handful of python functions into a Latch workflow"